spec/services_discover/index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'As a registered user' do
+  describe 'When I visit the Services Discover page' do
+    xit 'I see buttons to select what category to explore' do
```

Comment:
Can we add the Utilities for now here?

Reply:
I'll let Jake and Kevin know. I think they're working on that section :)